### PR TITLE
Use requests to query Ensembl API for the gene sequence and summary metadata

### PR DESCRIPTION
**Pull request by Lilithness**
Use requests library to fetch a DNA sequence, and look up the mutation's symbol in Ensembl.

### CODE REVIEW

File: functions.py [hunk 1]
```diff
@@ -1,5 +1,14 @@
 from info import *
 
+
+def fetch_seq():
+    url = "https://rest.ensembl.org/sequence/region/human/11: 5,225,464-5,229,395:-1"
+    r =requests.get(url, headers={ "Content-Type" : "text/plain"})
```
Comment (corvus-sapiens) [inline functions.py:6]: Can you please check if Ensembl API offers the FASTA format?

File: functions.py [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
-    r =requests.get(url, headers={ "Content-Type" : "text/plain"})
+    r = requests.get(url, headers={"Content-Type": "text/x-fasta"})
     r.raise_for_status()
-    f= open("sequence.txt", "w+")
-    f.write(r.text)
-    f.close()
-
+    with open("SickleCell.txt", "w+") as file:
+        sequence = r.text
+        file.write(sequence)
```
Comment (corvus-sapiens) [inline functions.py:14]: This can be a one-liner

File: functions.py [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
+    with open("SickleCell.txt", "r") as file:
+        temp = []
+        for i in file:
+            temp.append(i)
+    dna = ''.join(temp[1:])
```
Comment (corvus-sapiens) [inline functions.py:19]: Why not use list comprehension syntax?)

Comment (corvus-sapiens) [reply functions.py:19]: Also, this is definitely the case to use Biopython, -- but let us not add that dependency just yet.

**Review by corvus-sapiens — CHANGES_REQUESTED** [review]

File: functions.py [hunk 4]
```diff
@@ -10,12 +10,9 @@ def fetch_Seq(None):->str
...
     with open("SickleCell.txt", "w+") as file:
-        sequence = r.text
-        file.write(sequence)
+        file.write(r.text)
     with open("SickleCell.txt", "r") as file:
-        temp = []
-        for i in file:
-            temp.append(i)
+        temp = [i for i in file]
     dna = ''.join(temp[1:])
```
Comment (corvus-sapiens) [inline functions.py]: This, again, can be a one-liner. And also, put a comment explaining why we need that slicing here, please.

**Comment by Lilithness** [comment]
Use requests to query Ensembl API for the gene sequence and summary metadata

**Pull request merged**